src/js/components/DateInput/__tests__/DateInput-test.tsx

```diff
@@ -762,10 +762,9 @@ describe('DateInput', () => {
     fireEvent.change(getByPlaceholderText('mm/dd/yyyy-mm/dd/yyyy'), {
       target: { value: '07//2020-07/27/2021' },
     });
-    expect(onChange).toHaveNthReturnedWith(3, []);
```

Comment:
Could this assertion just be changed to `expect(onChange).toHaveNthReturnedWith(3, ['07/']);` instead of being removed?

Reply:
Ah okay I see